src/EntityFactory.cpp

```diff
@@ -394,6 +394,34 @@ namespace tjg {
         return tech17;
     }
 
+    std::shared_ptr<Entity> EntityFactory::MakeEntrance(const sf::Vector2f &a) {
+        // Create entrance entity.
+        auto entrance = std::make_shared<Entity>();
+
+        // Add location component
+        auto entrance_location = entrance->AddComponent<Location>(a.x, a.y);
+//        entrance_location->setRotation(calculateAngle(a, b));
```

Comment:
I think some of these comments can be removed?

Reply:
Yes, I removed them in my other branch for the exit.